pilot/pkg/bootstrap/server.go

```diff
@@ -216,7 +207,7 @@ func NewServer(args *PilotArgs, initFuncs ...func(*Server)) (*Server, error) {
 	}
 	// Initialize workload Trust Bundle before XDS Server
 	e.TrustBundle = s.workloadTrustBundle
-	s.XDSServer = xds.NewDiscoveryServer(e, args.Plugins, args.PodName, args.Namespace, args.RegistryOptions.KubeOptions.ClusterAliases)
```

Comment:
is it intentional to remove namespace?

Reply:
Yeah its not used (existing dead code). Already was cleaning up plugins so figured would do both